packages/orchestration/src/orchestration-util.ts

```diff
@@ -0,0 +1,57 @@
+import type {
+  AzureContentSafety,
+  DataRepositoryType,
+  DocumentGroundingFilter,
+  GroundingModuleConfig,
+  InputFilteringConfig,
+  OutputFilteringConfig
+} from './client/api/schema/index.js';
+
+/**
+ * Convenience function to create Azure content filters.
+ * @param filter - Filtering configuration for Azure filter. If skipped, the default Azure content filter configuration is used.
+ * @returns An object with the Azure filtering configuration.
+ */
+export function buildAzureContentFilter(
+  filter?: AzureContentSafety
+): InputFilteringConfig | OutputFilteringConfig {
+  if (filter && !Object.keys(filter).length) {
+    throw new Error('Filter property cannot be an empty object');
+  }
+  return {
+    filters: [
+      {
+        type: 'azure_content_safety',
+        ...(filter && { config: filter })
+      }
+    ]
+  };
+}
+
+export type DocumentGroundingFilterWrapper = Pick<DocumentGroundingFilter,
+  'id' | 'search_config' | 'data_repositories' | 'data_repository_metadata' | 'document_metadata' | 'chunk_metadata'> & {
```

Comment:
[pp] Do you think it would be helpful to have a helper function for setting all these three metadata fields?

Reply:
I am not aware what keys are supported, also couldn't find any documentation. I have asked Govind. Would be nicer if orchestration itself adds this in the spec.